src/components/DocumentationTopic/Summary/AvailabilityRange.vue

```diff
@@ -54,9 +54,9 @@ export default {
         platformName: name,
       } = this;
       return deprecatedAt ? (
-        `Introduced in ${name} ${introducedAt} and deprecated in ${name} ${deprecatedAt}`
+        this.$t('availability.introduced-and-deprecated', { name, introducedAt, deprecatedAt })
       ) : (
-        `Available on ${name} ${introducedAt} and later`
+        this.$t('availability.available-on', { name, introducedAt })
```

Comment:
Would `name` and `introducedAt` be localized?

Reply:
Name should come localised from the JSON file and `introducedAt` should be a string with number format, afaik